### DISCUSSION TITLE
How can I use this QA site?

### DISCUSSION THREAD

**Discussion by shanopaco** [HumanSupportRobot/QandA#3] [Q&A]
How can I use this QA site?

**Comment by shanopaco** [comment]
You can ask questions from "Issues" or "Discussions".

**Comment by hsr-admin** [comment]
For technical questions, please follow the STEP below to attempt to solve them.
[STEP 1] Investigation by yourself
Please check FAQ on hsr site to solve the problem by yourself.
[STEP2] Ask question to this QA site
Please register it as a new question at Discussions in this github repository.
Everybody can answer it. It is expected to solve problems by mutual cooperation among members.
